pkg/template/template.go

```diff
@@ -134,3 +136,29 @@ func (r ReleaseSuite) ComponentReleaseVersion(repo string) string {
 
 	return ""
 }
+
+func markdownHyperlinksToHTMLHyperlinks(sectionItem string) string {
+	linkTemplate := `<a href="%s">%s</a>`
+
+	markdownRegex := regexp.MustCompile(`\[(.*?)\]\((.*?)\)`)
+	nameRegex := regexp.MustCompile(`\[(.*)\]`)
+	urlRegex := regexp.MustCompile(`\((.*)\)`)
+
+	links := markdownRegex.FindAllString(sectionItem, -1)
+
+	for _, markdownLink := range links {
+		url := urlRegex.FindString(markdownLink)
+		url = strings.Replace(url, "(", "", 1)
+		url = strings.Replace(url, ")", "", 1)
+
+		name := nameRegex.FindString(markdownLink)
+		name = strings.Replace(name, "[", "", 1)
+		name = strings.Replace(name, "]", "", 1)
```

Comment:
I think this can be optimized with regex groups but no need to hold up the PR over it.